include/problem/MFEMProblem.h

```diff
@@ -147,6 +142,18 @@ class MFEMProblem : public ExternalProblem
   InputParameters addMFEMFESpaceFromMOOSEVariable(InputParameters & moosevar_params);
 
 protected:
+  /**
+   * Called internally by setMFEMVarData.
+   */
+  void setMFEMNodalVarData(MooseVariableFieldBase & moose_variable);
+  void setMFEMElementalVarData(MooseVariableFieldBase & moose_variable);
+
+  /**
+   * Called internally by setMooseVarData.
+   */
+  void setMooseNodalVarData(MooseVariableFieldBase & moose_variable);
+  void setMooseElementalVarData(MooseVariableFieldBase & moose_variable);
```

Comment:
MOOSE should be capitalised here to be consistent with `setMOOSEVarData` and `setMFEMVarData`